<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMainMenuH</class>
 <widget class="QWidget" name="FormMainMenuH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="ButtonBeR" name="btn_alarms">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Alarmes</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/alarmsslogs.png</normaloff>:/icons/alarmsslogs.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="bgIcon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/bt-blank.png</normaloff>
          <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
        </property>
        <property name="bgIconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="ButtonBeR" name="btn_products">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Produtos</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/prod.png</normaloff>:/icons/prod.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>38</height>
         </size>
        </property>
        <property name="bgIcon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/bt-blank.png</normaloff>
          <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
        </property>
        <property name="bgIconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ButtonBeR" name="btn_printer">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Impressora</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/printer.png</normaloff>:/icons/printer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="bgIcon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/bt-blank.png</normaloff>
          <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
        </property>
        <property name="bgIconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ButtonBeR" name="btn_config">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Config</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="bgIcon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/bt-blank.png</normaloff>
          <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
        </property>
        <property name="bgIconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="ButtonBeR" name="btn_export_data">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Exportar
Receitas</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/usb_pendrive.png</normaloff>:/icons/usb_pendrive.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>16</height>
         </size>
        </property>
        <property name="bgIcon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/bt-blank.png</normaloff>
          <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
        </property>
        <property name="bgIconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="ButtonBeR" name="btn_part11">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Relatório
Part 11</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/p11.png</normaloff>:/icons/p11.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>27</width>
          <height>32</height>
         </size>
        </property>
        <property name="bgIcon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/bt-blank.png</normaloff>
          <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
        </property>
        <property name="bgIconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ButtonBeR" name="btn_version">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Versão de
Software</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>27</width>
          <height>32</height>
         </size>
        </property>
        <property name="bgIcon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/bt-blank.png</normaloff>
          <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
        </property>
        <property name="bgIconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="ButtonBeR" name="btn_learning">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Ensinar OCR</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/book-alt-2.png</normaloff>:/icons/book-alt-2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="bgIcon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/bt-blank.png</normaloff>
          <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
        </property>
        <property name="bgIconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_login">
     <property name="styleSheet">
      <string notr="true">image: url(:/images/bg.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonBeR</class>
   <extends>QPushButton</extends>
   <header>buttonber.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
